profilers/network_resource_profiler_mulhome/home/central_scheduler.py

```diff
@@ -419,20 +424,35 @@ def main():
     config = configparser.ConfigParser()
     config.read(INI_PATH)
 
-    global MONGO_DOCKER, FLASK_SVC, FLASK_DOCKER, num_retries, username, password, ssh_port, HOME_IP
+    global MONGO_DOCKER, FLASK_SVC, FLASK_DOCKER, num_retries, username, password, ssh_port, HOME_IP, min_file, max_file
 
     username    = config['AUTH']['USERNAME']
     password    = config['AUTH']['PASSWORD']
     ssh_port    = int(config['PORT']['SSH_SVC'])
     num_retries = int(config['OTHER']['SSH_RETRY_NUM'])
+    min_file = int(math.log10(int(config['OTHER']['MIN_SIZE'])))
+    max_file = int(math.log10(int(config['OTHER']['MAX_SIZE'])))
+
+    global file_size,num_files,cur_idx
```

Comment:
Please add spaces after all commas. It's difficult to read code without the spaces, and it's also a PEP standard.

Spaces should be before and after all math operators (like  a + or -) except when overriding a default input argument to a function for example myFunc(arg=10, someString="hi").